lib/templates/pobject/system.export.templates.phtml

```diff
@@ -28,9 +28,10 @@ if (! ( $templates && is_array($templates) && !empty($templates) ) && !$ARCurren
 				$templates=$this->data->config->pinp;
 				reset($templates);
 				display("\n	exporting/importing templates\n");
-				while (is_array($templates) && (list($type, $functions)=each($templates))) {
-					while (is_array($functions) && (list($function, $languages)=each($functions))) {
-						while (is_array($languages) && (list($language, $file)=each($languages))) {
+
```

Comment:
I don't know how important the `is_array($templates)` in the original code is, but it _might_ cause trouble being absent in the change.